src/pages/basic_data/app_admin/NviPeriodsPage.tsx

```diff
@@ -32,14 +33,10 @@ export const NviPeriodsPage = () => {
         <List disablePadding>
           {sortedPeriods.map((nviPeriod) => {
             const startDateString = nviPeriod.startDate
-              ? `${new Date(nviPeriod.startDate).toLocaleDateString()} (${new Date(
-                  nviPeriod.startDate
-                ).toLocaleTimeString()})`
+              ? `${toDateString(nviPeriod.startDate)} (${new Date(nviPeriod.startDate).toLocaleTimeString()})`
```

Comment:
Burde også klokkelsett gjøres om til norsk format? Akkurat nå står det klokkelsett med AM/PM.

Reply:
Tenkte på det, men vurderte det som at var like greit å la det ligge siden problemet med dato var at de ble vist ulikt ulike plasse, noe som ikke er tilfelle for klokkeslett.
Syns du det er bedre sikre norsk der også?

Det samme gjelder `toLocaleString()` på tall også, hvor jeg heller ikke tvinger bruker over til norsk. Norsk browser vil feks få `10 222,50` mens amerikansk vil få `10,222.50`. Kunne tvingt alt over på norsk, men usikker selv om det skaper mer klarhet eller forvirring 😅 

Heller litt mot å bruke default funksjonalitet på alt som ingen melder som bug (som at dato ble vist på ulike format)

Reply:
Kan godt bruke default funksjonalitet for min del. Alle forstår jo AM/PM, så det skaper jo ikke forvirring 👍

Reply:
Jeg skjønner ikke AM og PM, blander alltid med After/Past og Midday/Midnigth, så AM og PM kan jo være hva som helst :D